packages/design/src/icons/iconMap.ts

```diff
@@ -5,552 +5,409 @@
+    "filter": [
+      "M682.667 384c55.731 0 103.147-35.619 120.717-85.333h49.95c23.565 0 42.667-19.103 42.667-42.667s-19.102-42.667-42.667-42.667h-49.95c-17.57-49.715-64.986-85.333-120.717-85.333s-103.147 35.619-120.717 85.333h-391.283c-23.564 0-42.667 19.103-42.667 42.667s19.103 42.667 42.667 42.667h391.283c17.57 49.715 64.986 85.333 120.717 85.333zM725.333 256c0 23.564-19.102 42.667-42.667 42.667s-42.667-19.103-42.667-42.667c0-23.564 19.102-42.667 42.667-42.667s42.667 19.103 42.667 42.667z",
+      "M376.716 554.667h476.617c23.565 0 42.667-19.102 42.667-42.667s-19.102-42.667-42.667-42.667h-476.617c-17.571-49.715-64.984-85.333-120.716-85.333-70.692 0-128 57.306-128 128s57.308 128 128 128c55.732 0 103.145-35.618 120.716-85.333zM298.667 512c0 23.565-19.103 42.667-42.667 42.667s-42.667-19.102-42.667-42.667c0-23.565 19.103-42.667 42.667-42.667s42.667 19.102 42.667 42.667z",
+      "M348.617 810.667c17.571 49.715 64.984 85.333 120.716 85.333s103.147-35.618 120.717-85.333h263.283c23.565 0 42.667-19.102 42.667-42.667s-19.102-42.667-42.667-42.667h-263.283c-17.57-49.715-64.986-85.333-120.717-85.333s-103.145 35.618-120.716 85.333h-177.95c-23.564 0-42.667 19.102-42.667 42.667s19.103 42.667 42.667 42.667h177.95zM469.333 810.667c-23.565 0-42.667-19.102-42.667-42.667s19.102-42.667 42.667-42.667c23.565 0 42.667 19.102 42.667 42.667s-19.102 42.667-42.667 42.667z"
     ],
-    "gift": [
-      "M256 234.667c0-82.475 66.859-149.333 149.333-149.333 50.479 0 95.113 25.048 122.142 63.394 20.087-13.108 44.083-20.727 69.858-20.727 70.694 0 128 57.308 128 128 0 14.961-2.569 29.321-7.283 42.667h177.95c23.565 0 42.667 19.103 42.667 42.667v170.667c0 23.565-19.102 42.667-42.667 42.667h-42.667v298.667c0 47.13-38.204 85.333-85.333 85.333h-512c-47.128 0-85.333-38.204-85.333-85.333v-298.667h-42.667c-23.564 0-42.667-19.102-42.667-42.667v-170.667c0-23.564 19.103-42.667 42.667-42.667h142.371c-9.215-19.398-14.371-41.097-14.371-64zM554.667 853.333h213.333v-298.667h-213.333v298.667zM469.333 554.667h-213.333v298.667h213.333v-298.667zM554.667 469.333h298.667v-85.333h-298.667v85.333zM469.333 384h-298.667v85.333h298.667v-85.333zM469.333 298.667v-64c0-35.346-28.655-64-64-64-35.346 0-64 28.654-64 64s28.654 64 64 64h64zM554.667 256v42.667h42.667c23.565 0 42.667-19.103 42.667-42.667s-19.102-42.667-42.667-42.667c-23.565 0-42.667 19.103-42.667 42.667z"
+    "back-to-today": [
+      "M682.667 128c-23.565 0-42.667 19.103-42.667 42.667v42.667h-256v-42.667c0-23.564-19.103-42.667-42.667-42.667s-42.667 19.103-42.667 42.667v42.667h-85.333c-47.128 0-85.333 38.205-85.333 85.333v512c0 47.13 38.205 85.333 85.333 85.333h597.333c47.13 0 85.333-38.204 85.333-85.333v-512c0-47.128-38.204-85.333-85.333-85.333h-85.333v-42.667c0-23.564-19.102-42.667-42.667-42.667zM810.667 298.667v85.333h-597.333v-85.333h597.333zM213.333 469.333h597.333v341.333h-597.333v-341.333zM704 640c-35.345 0-64-28.655-64-64s28.655-64 64-64c35.345 0 64 28.655 64 64s-28.655 64-64 64z"
     ],
     "recurring": [
```

Comment:
The modifications to this file are extensive - much more than just adding the four or five new icons mentioned in the PR description. Are all the changes to the existing icons expected?

Reply:
This looks unexpected, for reference here's a previous icon update that only touches the vectors of existing icons:

https://github.com/GetJobber/atlantis/pull/860/files

That said there were some issues on the IcoMoon side of generating the updated icons so I wouldn't be surprised if something ended up being re-drawn :thinking:

The icons themselves look as expected when I can convince this to build locally:
![image](https://user-images.githubusercontent.com/39704901/195642638-5db871a2-4553-4898-b420-c498ef671cea.png)

Reply:
This could explain the abundance of snapshot failures as the icons are being redrawn marginally differently, even when the "right icon" is being rendered.

Reply:
Thank you @jcw- for the feedback. It's strange that these changes are unexpected, as I only ran the command `npm run convertIcoMoon` on the json file in the `jobberIcons` folder as instructed in the documentation. There was also a similar change to the `package-lock.json` file in a previous PR that I found so I thought that I should push the `package-lock.json` file as well. Should it be removed?

Reply:
i probably would (looks like you already have!), to keep this PR more straight forward / easier to test